src/Listeners/AddUserBioAttribute.php

```diff
@@ -39,9 +45,21 @@ public function __invoke(UserSerializer $serializer, User $user, array $attribut
     {
         $actor = $serializer->getActor();
 
+        $bio = $user->bio ?? '';
+        $isXML = str_starts_with($bio, '<');
+
         if ($actor->can('viewBio', $user)) {
+            if ($isXML) {
+                $attributes['bioHtml'] = $this->formatter->render($bio);
+
+                if ($actor->id === $user->id) {
```

Comment:
Shouldn't this use a gated ability? Moderators won't be able to access the editable version of the bio otherwise :thinking: